x/incentives/keeper/distribute.go

```diff
@@ -283,23 +283,36 @@ func (k Keeper) AllocateAcrossGauges(ctx sdk.Context) error {
 			return err
 		}
 
+		// TODO: handle this error quietly
```

Comment:
The TODOs in this function are left for reference in #6402 once this function is implemented to support volume splitting and other group gauge changes. Happy to extract this if we want to keep the function clean on main, but I think having these comments here will make it much easier to make the necessary changes later